### DISCUSSION TITLE
Cannot enable GitHub CodeSpaces or find out status of upgrade request

### DISCUSSION THREAD

**Discussion by lbang-hmc** [community/community#57107] [GitHub Education]
### Hi everyone,

I cannot figure out how to enable CodeSpaces in my GitHub classroom. I suspect that this is because the organization is not yet upgraded, but I can't tell what is the status of my upgrade. Here is a video that I hope succinctly describes the issue:

https://github.com/community/community/assets/46112584/6e0b0509-1048-4b4e-a332-f370bedabdb0

**Comment by MUScphillips** [comment]
Are you still having this issue? I'm running into the same problem.

**Comment by lbang-hmc** [comment]
The TLDR is that I just waited a while and eventually it went through. It took several days I believe. 

Overall, I find that the instructions, links, and menus for all of these actions take you down incorrect workflow paths. Furthermore, the inability to check on the status of an upgrade request is frustrating, as we cannot tell if it has been processed or not, or when it might be completed.
